src/migrations/20230807205306-add-uuid-to-accounts.ts

```diff
@@ -0,0 +1,44 @@
+/* eslint @typescript-eslint/ban-ts-comment: "off" */
+// @ts-nocheck
+/* eslint @typescript-eslint/no-var-requires: "off" */
+const getUuidByString = require("uuid-by-string")
+
+async function migrateAccounts(db, batchSize = 100) {
+  const cursor = db.collection("accounts").find({ uuid: { $exists: false } })
+
+  let batchCount = 0
+  while (await cursor.hasNext()) {
+    const batchUpdates = []
+    for (let i = 0; i < batchSize && (await cursor.hasNext()); i++) {
+      const account = await cursor.next()
+      const uuid = getUuidByString(account._id.toString())
+      batchUpdates.push({
+        updateOne: {
+          filter: { _id: account._id },
+          update: { $set: { uuid } },
```

Comment:
we call it `id` for the Wallet. I think we should keep the same pattern here ( `_id` for internal mongo object and `id` for the external identifier)